pkg/profile/profile.go

```diff
@@ -21,13 +21,38 @@ type Profile struct {
 }
 
 func (p Profile) AsLog() any {
+	var name, email, phone, nin string 
+	if len(p.Name) < 3 {
+		name = p.Name
+	} else {
+		name = p.Name[:3]
+	}
+
+	if len(p.Email) < 3 {
+		email = p.Email
+	} else {
+		email = p.Email[:3]
+	}
+
+	if len(p.Phone) < 3 {
+		phone = p.Phone
+	} else {
+		phone = p.Phone[:3]
+	}
+
+	if len(p.NIN) < 3 {
+		nin = p.NIN
+	} else {
+		nin = p.NIN[:3]
+	}
```

Comment:
It might be beneficial to streamline these conditions for better readability :

```go
    if len(p.Name) >= 3 {
        p.Name = p.Name[:3]
    }
    // .....
    return Profile{
        Name:    p.Name + "***",
        // .....
    }
```

Reply:
Cool, many thanks mas @nasrulfuad .